Docs/source/theory/collisions.rst

```diff
@@ -3,18 +3,36 @@
 Collisions
 ==========
 
-Monte Carlo Collisions
-----------------------
+WarpX includes several different models to capture collisional processes
+including collisions between kinetic particles (Coulomb collisions, DSMC,
+nuclear fusion) as well as collisions between kinetic particles and a fluid
```

Comment:
```suggestion
nuclear fusion) as well as collisions between kinetic particles and a fixed (i.e. non-evolving)
```
The word `fluid` here might be confusing ; some might understand this as interacting with the hybrid fluid-kinetic solver, or the fluid species in WarpX.